astoria.toml

```diff
@@ -14,4 +14,7 @@ bridge = "br0"
 enable_wpa3 = false
 
 [system]
-cache_dir = ".cache_dir"  # Use a directory in /var for production
\ No newline at end of file
+cache_dir = ".cache_dir"  # Use a directory in /var for production
+initial_log_lines = [
```

Comment:
Given TOML supports multi-line strings, should it just be that instead?

On the one hand it makes defining the config much cleaner, although precautions may need to be taken around ensuring platform-specific line endings are converted as necessary.

Reply:
We'd need to convert it into a list in `astprocd` anyway as the logging code only handles single lines at a time (This is so that lines are logged onto MQTT for the web interface etc.)